tools/lsmcli/cmdline.py

```diff
@@ -1488,7 +1488,8 @@ def volume_replicate_range(self, args):
 
         ranges = []
         for b in range(len(src_starts)):
-            ranges.append(BlockRange(src_starts[b], dst_starts[b], counts[b]))
+            ranges.append(BlockRange(int(src_starts[b]), int(dst_starts[b]),
+                                     int(counts[b])))
```

Comment:
C is treating this as an unsigned 64 bit integer.  When running python2 on 32bit hardware a python `int` is limited to 2**32-1.  We should be casting these to python `long`.